### ISSUE TITLE
False Error in Mutation

### ISSUE THREAD

**Issue by trbngr** [jimkyndemeyer/js-graphql-intellij-plugin#23]
The following syntax is marked as an error in `Mutation.getMutation()`.

```javascript
  getMutation() {
    return Relay.QL`mutation{changeConference}`;
  }
```

![image](https://cloud.githubusercontent.com/assets/364786/18189032/e166c9ca-706b-11e6-91bf-eed32606c3b7.png)

**Comment by jimkyndemeyer** [comment]
Hi Chris.

Thanks for catching this. I'll include a fix for the next release.

/Jim

**Comment by jimkyndemeyer** [comment]
Fixed with [v1.2.1](https://github.com/jimkyndemeyer/js-graphql-language-service/releases/tag/v1.2.1) of js-graphql-language-service which is included in the next release.
